docs/guides/update-guide/connectors/010-to-020.md

```diff
@@ -159,10 +159,10 @@ If using the
 [pre-packaged runtime environment](/components/integration-framework/connectors/custom-built-connectors/connector-sdk.md#pre-packaged-runtime-environment)
 that comes with the SDK does not fit your use case, you can create a custom runtime environment.
 
-In version 0.2.0 of the SDK, the names of the following classes need to be adjusted:
+In version 0.2.0 of the [job worker connector run-time](https://github.com/camunda/connector-sdk/tree/main/runtime-job-worker), the following changes are required:
```

Comment:
So far, we mostly link to other docs pages. I think it's easier to connect the dots, I think, i.e. "Did I do this? Ah yeah, this section of the docs I followed when I implemented my own runtime environment".
```suggestion
With version 0.2.0 of the [job worker runtime environment](/components/integration-framework/connectors/custom-built-connectors/connector-sdk.md#connector-job-handler), you need to make the following changes:
```

Reply:
Plus, can we add this to the 8.0 docs here as well?
And this needs to be adjusted in the `connector-sdk.md` as well in the section [Connector job handler](https://docs.camunda.io/docs/components/integration-framework/connectors/custom-built-connectors/connector-sdk/#connector-job-handler) (also both in next and 8.0)

Reply:
How shall I add it to 8.0 as well? Plz help :see_no_evil:

Reply:
Fair enough. There are other changed files here in the PR that are located in `versioned-docs/version-8.0`, like the [SDK guide](https://github.com/camunda/camunda-platform-docs/pull/1255/files#diff-0a5a5fc9921d584ec150ba65d9d7b72dd1f2ba86bb91ed146fa39ff6e4c3fe78).

Reply:
Will update. Thanks!